src/views/GovernanceTaskList/index.scss

```diff
@@ -0,0 +1,88 @@
+.governance-task-list {
+  // Keep heading and order numbers the same
+  &__task-heading,
+  &__item:before {
+    font-size: 1.375em;
+  }
+
+  &__task-list {
+    padding-left: 0;
+    counter-reset: primaryList secondaryList 3;
```

Comment:
`primaryList` counter starts at 0 and `secondaryList` counter starts at 3; meaning the first number it shows is 4.